cve_bin_tool/extractor.py

```diff
@@ -109,6 +112,26 @@ async def extract_file_rpm(self, filename, extraction_path):
                     return 1
         return 0
 
+    async def extract_file_zst(self, filename, extraction_path):
+        """Extract zstd compressed files"""
+
+        if await aio_inpath("zstd"):
+            tar_path = os.path.join(extraction_path, "data.tar")
+            self.logger.warning(tar_path)
```

Comment:
I'm guessing you left it for debugging while writing code.

Reply:
Yes this was debugging, sorry about that!

Reply:
Alright so for the moment, I will wait before implementing the 7z method and move on to extracting .pkg files instead!

Reply:
Notes from meeting:
- generally, fewer dependencies is better
- algorithms in extractors can have patent issues, so people may be extra concerned about extra packages
- sounds like this can all be done with zstandard so maybe do that?